### PR TITLE
Fix Issue  #10410

### PR DESCRIPTION
**Pull request by KierenLWoods**
## Proposed changes

 This change fixes bug #10410,  There was an issue whereby if you passed a suite parameter and an empty spec file, additional spec files where incorrectly run. This was caused when the isSpecParamPassed property was set to true as the empty array was still setting the property to true. This change adds an additional check to make sure this is only set to true when the this._config.spec actually contains a spec file.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/wdio-config/src/lib/ConfigParser.ts [hunk 1]
```diff
@@ -254,7 +254,7 @@ export default class ConfigParser {
      * attributes from CLI, config and capabilities
      */
     getSpecs(capSpecs?: Spec[], capExclude?: Spec[]) {
-        const isSpecParamPassed = Array.isArray(this._config.spec)
+        const isSpecParamPassed = Array.isArray(this._config.spec) && Array(this._config.spec)[0]?.length !== 0
```
Comment (erwinheitzman) [inline packages/wdio-config/src/lib/ConfigParser.ts:257]:
```suggestion
        const isSpecParamPassed = Array.isArray(this._config.spec) && this._config.spec.length >= 1
```

**Review by erwinheitzman — CHANGES_REQUESTED** [review]
Can you please fix the issue template formatting and fill in the description with a link to the issue it fixes? You can link it by using a hashtag followed by the issue number (found in the url), if done it will link the issue and automatically close it when this is merged.

**Comment by erwinheitzman** [comment]
Thanks for the PR 👍

**Comment by erwinheitzman** [comment]
As a first time contributor you'll have to sign the CLA to be able to make contributions to the project. Could you please sign it?

**Comment by KierenLWoods** [comment]
@erwinheitzman created a new PR as was having issues resolving the initial easy CLA failures https://github.com/webdriverio/webdriverio/pull/11271

**Pull request closed without merging**